community/shell/pom.xml

```diff
@@ -168,6 +168,19 @@ the relevant Commercial Agreement.
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.neo4j</groupId>
+      <artifactId>neo4j-bolt</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Add bolt as a test dependency, so that we can test the effect of having it on the classpath.